src/pages/overlay/components/buttons/buttons.module.scss

```
@@ -7,27 +7,31 @@
   margin: 3rem 0 0;
   z-index: 1; // stay above the popups
 
-  button {
+  .btn,
+  .tooltip {
     display: flex;
     justify-content: center;
     align-items: center;
```

```suggestion
  .btn {
```
No point having two classes doing the same thing + moving the flex alignment to the button, as the image is inside that